mlflow/pyfunc/__init__.py

```diff
@@ -724,6 +729,80 @@ def load_model(
     return PyFuncModel(model_meta=model_meta, model_impl=model_impl)
 
 
+def _download_model_conda_env(model_uri):
+    conda_yml_file_name = _get_flavor_configuration_from_uri(model_uri, FLAVOR_NAME)[ENV]
+    return _download_artifact_from_uri(append_to_uri_path(model_uri, conda_yml_file_name))
+
+
+def get_model_dependencies(model_uri, format="pip"):  # pylint: disable=redefined-builtin
+    """
+    Given a model URI, and format ("pip" or "conda", default value "pip"),
+    return the downloaded dependency file path.
+    If "pip" format specified but model does not have "requirements.txt" file,
+    fallback to parse the pip section of the model's "conda.yaml" and ignore other
+    non-pip dependencies in the "conda.yaml" file.
```

Comment:
```suggestion
    non-pip dependencies in the "conda.yaml" file.
    
    :param: ...
```

Can you add parame descriptions if this is a public API?